common/src/main/scala/org/apache/celeborn/common/protocol/message/ControlMessages.scala

```diff
@@ -1208,11 +1211,16 @@ object ControlMessages extends Logging {
 
       case HEARTBEAT_FROM_APPLICATION_VALUE =>
         val pbHeartbeatFromApplication = PbHeartbeatFromApplication.parseFrom(message.getPayload)
+        val shuffleFallbackCountMap = new util.HashMap[String, java.lang.Long]()
+        pbHeartbeatFromApplication.getShuffleFallbackCountsMap.asScala.foreach { entry =>
+          shuffleFallbackCountMap.put(entry._1, entry._2)
+        }
```

Comment:
can we use the `pbHeartbeatFromApplication.getShuffleFallbackCountsMap` directly?

or putAll to `shuffleFallbackCountMap`?